docs/source/getting_started.rst

```diff
@@ -14,9 +14,9 @@ FEniCS can be installed with Anaconda on MacOs and Linux.
 In order to use the Anaconda distribution on Windows, use the `Windows Subsystem for Linux <https://learn.microsoft.com/en-us/windows/wsl/install>`_. 
 First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands 
```

Comment:
```suggestion
First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands::
```

Just look at the other code blocks, you need to add :: to the paragraph before the code block